docs/blog/2018-07-07-graphic-design-class/index.md

```diff
@@ -79,7 +81,9 @@ One thing they wanted more of:
 
 They suggest that a link be added to the home page of Gatsby pointing to it and all sites with Git repos should be searchable first.
 
-> _NOTE_: His students are in luck! The [Starter Showcase](/starters/?sort=stars) (see [PR 6113](https://github.com/gatsbyjs/gatsby/pull/6113)) and [Site Showcase](/showcase/) ([see relevant issues here](https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+site+showcase)) are nearly done.
+<pullquote>
+_NOTE_: His students are in luck! The [Starter Showcase](/starters/?sort=stars) (see [PR 6113](https://github.com/gatsbyjs/gatsby/pull/6113)) and [Site Showcase](/showcase/) ([see relevant issues here](https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+site+showcase)) are nearly done.
```

Comment:
Markdown (probably?) won't be supported in these. I'd actually revert this, since it doesn't seem to be a great fit for a pullquote.

Reply:
Yeah. I felt that too. Will update it